src/features/filterPosts/filterPostsSlice.ts

```diff
@@ -0,0 +1,49 @@
+import { createSlice, type PayloadAction } from "@reduxjs/toolkit";
+import { stat } from "fs";
+import { IPost } from "../posts/postsSlice";
+
+export type filt = {
```

Comment:
Не сокращай названия
```suggestion
export type filteredPosts = {
```